test/box/tuple.result

```diff
@@ -364,6 +364,34 @@ t[nil]
 - null
 ...
 --------------------------------------------------------------------------------
+-- test tuple:format
+--------------------------------------------------------------------------------
+f = box.tuple.format.new({{'id', 'number'}, {'name', 'string'}})
+---
```

Comment:
It looks like it's better to test this feature using unit tests, see example:
https://github.com/tarantool/tarantool/pull/8549/

tests with the output have not been updated for a long time, there is unit-tests prefered at now.

Reply:
Added new tests into `test/app-luatest` directory.

Reply:
So we could remove the tests now.

Reply:
So we could remove the tests now.

Reply:
Ok, updated.